importer/MtDe.py

```diff
@@ -0,0 +1,96 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class MtDe(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Use the 'gemeinde' field as
+        municipality.
+        """
+        municip_raw = self.gemeinde
+        municip_q = utils.q_from_wikipedia("de", municip_raw)
+        if municip_q:
+            self.add_statement("located_adm", municip_q)
+        else:
+            self.add_to_report("gemeinde", self.gemeinde, "located_adm")
+
+    def update_labels(self):
+        labels = {
+            "de": self.name_de,
+            "en": self.name_en,
+            "mt": self.name_mt
+        }
+        for language in labels:
+            self.add_label(language, labels[language])
+
+    def update_descriptions(self):
+        """
+        Set the descriptions in English.
+
+        Try to add the place name. Since the municipality
+        names are titles of articles, sometimes they
+        have the form of "$municipality (Malta)", in which
+        case remove the bracketed part.
+        """
+        placename = "Malta"
+        description = "cultural property in {}"
+        if self.has_non_empty_attribute("gemeinde"):
+            municip_base = utils.get_rid_of_brackets(self.gemeinde)
+            placename = "{}, {}".format(municip_base, "Malta")
+        self.add_description("en", description.format(placename))
+
+    def set_monuments_all_id(self):
+        self.monuments_all_id = self.inventarnummer
+
+    def set_heritage_id(self):
+        self.add_statement("malta_monument_id", self.inventarnummer)
```

Comment:
the number needs to be zero-padded to 5 digits per [P1799](https://www.wikidata.org/wiki/Property:P1799)

Reply:
Done.